### ISSUE TITLE
Aspekta Variable Font woff2 weight 400 does not handle left and right quote glyphs

### ISSUE THREAD

**Issue by scottykaye** [ivodolenc/aspekta#17]
### Steps to Reproduce

- Use aspekta variable font at weight 400 on a site
- use left “ and right  ” quote marks (in our use case, regular quotes " are updated to left and right quotes by notion and displayed that way on site)
- see that at font size 400 the quotes are a single foot instead of double for only weight 400

### Current Behavior

You can see at font weight 300 and 500, left and right quotes are treated as double correctly, but not correctly at 400.

<img width="377" alt="Screenshot 2024-06-17 at 3 19 48 PM" src="https://github.com/ivodolenc/aspekta/assets/5076841/af636658-a560-412b-9272-e4bef444cbc0">

### Expected Behavior

Expected for left and right quotes to output correctly as double quotes

### Additional Details

Heres a reproduction in [codepen](https://codepen.io/scottykaye/pen/xxNpoqK). It's possible the URL goes away when we push up a deploy, I will do my best to keep it in sync.

Thanks!

### Version

2.0.0

**Comment by impressiver** [comment]
so weird

**Comment by ivodolenc** [comment]
Hi, yes, strange behavior.

It works as expected in desktop apps, so it's probably rendering issue in the `woff2` format or in browsers.

Try using **Aspekta Variable** in `ttf` format for now, I just tested it and it works ok.

### [Download Official Aspekta Typeface](https://github.com/ivodolenc/aspekta/releases)

---

This quick fix works too in the `woff2` format:

```html
<!-- set `font-weight` to 401 or 399 -->
<li style="font-weight: 401">
    Font Weight 400: Left quote: “ Right Quote: ”
</li>
```

**Comment by scottykaye** [comment]
Hey, thanks for such a quick response! 

Leaning toward going with the 401 solution only because the size of the font as woff2 is 29KB vs TTF is 68KB so we get a bit of a performance gain from the woff2.

This should work fine for us. Curious if you think this is something that may be fixed in the future?

**Comment by ivodolenc** [comment]
Yes, it needs to be fixed in future releases, but as I said, since it works fine in desktop apps and in `ttf` format, it's most likely a problem in `woff2` format. 

The mentioned quotes are built from smart components so they should display normal.

**Comment by scottykaye** [comment]
Thanks so much!

**Comment by ivodolenc** [comment]
Np, tnx for reporting 👍🏻
